misc/config_tools/schema/checks/vuart_config.xsd

```diff
@@ -15,7 +15,9 @@
   </xs:assert>
 
   <xs:assert test="every $io_port in /acrn-config//endpoint/io_port satisfies
```

Comment:
You can apply the check to the XPATH which `io_port` enumerates, i.e., `/acrn-config//endpoint[vm_name != '']/io_port`

Also you may want to update the XPATH used in `count` as well since you do not want to count ports for VMs without a name.